### PR TITLE
ci: add vars.APPLICATION_ID and vars.APPLICATION_NAME

### PR DESCRIPTION
**Pull request by longhuan1999**
Add vars.APPLICATION_ID and vars.APPLICATION_NAME to determine whether to customize the package name and app name of the manager

### CODE REVIEW

File: manager/build.gradle.kts [hunk 1]
```diff
@@ -89,6 +89,24 @@ fun Project.configureBaseExtension() {
...
+            prop.load(signFile.inputStream())
+            var APPLICATION_ID = prop.getProperty("APPLICATION_ID")
+            var APPLICATION_NAME = prop.getProperty("APPLICATION_ID")
+            if (APPLICATION_ID.isBlank()){
+                APPLICATION_ID = "me.weishu.kernelsu"
+            }
+            if (APPLICATION_NAME.isBlank()){
+                APPLICATION_NAME = "KernelSU"
+            }
+            defaultConfig.applicationId = prop.getProperty("APPLICATION_ID")
```
Comment (tiann) [inline manager/build.gradle.kts:103]: it should be `defaultConfig.applicationId = APPLICATION_ID`

File: manager/build.gradle.kts [hunk 2]
```diff
@@ -89,6 +89,24 @@ fun Project.configureBaseExtension() {
...
+            defaultConfig.resValue("string", "app_name", prop.getProperty("APPLICATION_NAME"))
```
Comment (tiann) [inline manager/build.gradle.kts:104]: should use `APPLICATION_NAME` instead.

File: manager/build.gradle.kts [hunk 3]
```diff
@@ -89,6 +89,24 @@ fun Project.configureBaseExtension() {
             signingConfigs["debug"]
         }
 
+        if (signFile.canRead()) {
+            val prop = Properties()
+            prop.load(signFile.inputStream())
+            var APPLICATION_ID = prop.getProperty("APPLICATION_ID")
+            var APPLICATION_NAME = prop.getProperty("APPLICATION_ID")
+            if (APPLICATION_ID.isBlank()){
+                APPLICATION_ID = "me.weishu.kernelsu"
```
Comment (tiann) [inline manager/build.gradle.kts:98]: better to use a variable such as `DEFAULT_APPLICATION_ID`, because it is used later again.

**Review by tiann — CHANGES_REQUESTED** [review]

File: manager/build.gradle.kts (hunk 1 above)
Comment (longhuan1999) [reply manager/build.gradle.kts]: Sorry, I was careless

File: manager/build.gradle.kts [hunk 4]
```diff
@@ -89,6 +89,27 @@ fun Project.configureBaseExtension() {
             signingConfigs["debug"]
         }
 
+        val DEFAULT_APPLICATION_ID = "me.weishu.kernelsu"
+        val DEFAULT_APPLICATION_NAME = "KernelSU"
+
+        if (signFile.canRead()) {
+            val prop = Properties()
+            prop.load(signFile.inputStream())
+            var APPLICATION_ID = prop.getProperty("APPLICATION_ID")
```
Comment (tiann) [inline manager/build.gradle.kts:98]: There is an overload method:

```
    public String getProperty(String key, String defaultValue) {
        String val = getProperty(key);
        return (val == null) ? defaultValue : val;
    }
```

So i think `prop.getProperty("APPLICATION_ID", DEFAULT_APPLICATION_ID)` is better.

**Review by tiann — CHANGES_REQUESTED** [review]

**Pull request closed without merging**